src/strategies/planet-finance-v2/index.ts

```diff
@@ -98,35 +151,60 @@ export async function strategy(
     { blockTag }
   );
 
+  // get all results
   const result = await Promise.all([
     score,
     usergAquaBalInAquaInfinityVault,
     usersNewAquaBnbVaultBalances,
-    usersAquaInLending
+    usersAquaInLending,
+    usersNewAquaBtcbVaultBalances,
+    usersNewAquaPoolBalances,
+    usersEarningSharesInAquaBtcbPool
```

Comment:
This will cross our limit of 5 network calls, refer to https://github.com/snapshot-labs/snapshot-strategies?tab=readme-ov-file#code